PyRDF/__init__.py

```diff
@@ -50,23 +51,14 @@ def use(backend_name, conf={}):
             necessary configuration parameters. Its default value is an empty
             dictionary {}.
     """
-    future_backends = [
-        "dask"
-    ]
 
     global current_backend
 
-    if backend_name in future_backends:
-        msg = "This backend environment will be considered in the future !"
-        raise NotImplementedError(msg)
-    elif backend_name == "local":
-        current_backend = Local(conf)
-    elif backend_name == "spark":
-        from PyRDF.backend.Spark import Spark
-        current_backend = Spark(conf)
+    if backend_name == "spark":
+        current_backend = Spark.Spark(conf)
     else:
-        msg = "Incorrect backend environment \"{}\"".format(backend_name)
-        raise Exception(msg)
+        raise NotImplementedError(
+            "The selected backend has not been implemented yet.")
```

Comment:
I'd say which backend: `Backend {} is not implemented`.format....

Reply:
Let's keep in mind that the `use` function will be removed with  #110

Reply:
Yes indeed